docs/pyboard/hardware/index.rst

```diff
@@ -3,9 +3,12 @@ The pyboard hardware
 
 For the pyboard:
 
-* `PYBv1.0 schematics and layout <http://micropython.org/resources/PYBv10b.pdf>`_ (2.4MiB PDF)
-* `PYBv1.0 metric dimensions <http://micropython.org/resources/PYBv10b-metric-dimensions.pdf>`_ (360KiB PDF)
-* `PYBv1.0 imperial dimensions <http://micropython.org/resources/PYBv10b-imperial-dimensions.pdf>`_ (360KiB PDF)
+* v1.1
+    * `PYBv1.1 schematics <https://micropython.org/resources/PYBv11-schematics.pdf>`_ (261KiB PDF)
```

Comment:
Can you please update this to PYBv11.pdf?  And a description could be "schematics, layout and dimensions".

Reply:
Fixed in 9827cf9e40a2ffb2aef8efbc21d076de7be72553